instaloader/instaloader.py

```diff
@@ -718,36 +718,37 @@ def _all_already_downloaded(path_base, is_videos_enumerated) -> bool:
         downloaded = True
         if post.typename == 'GraphSidecar':
             if self.download_pictures or self.download_videos:
-                if not _all_already_downloaded(
-                        filename_template, enumerate(
-                            (post.get_is_videos()[i]
-                             for i in range(self.slide_start % post.mediacount, self.slide_end % post.mediacount + 1)),
-                            start=self.slide_start % post.mediacount + 1
-                        )
-                ):
-                    for edge_number, sidecar_node in enumerate(
-                            post.get_sidecar_nodes(self.slide_start, self.slide_end),
-                            start=self.slide_start % post.mediacount + 1
+                if not (post.mediacount) == 0: # make sure post hasnt been edited / removed
```

Comment:
Thanks for fixing this issue!

I think it would be nice to group these two ifs together, to avoid nesting and increase readability, like so

```python
            if (self.download_pictures or self.download_videos) and post.mediacount > 0:
```